workflow/scripts/predict_affinities.py

```
@@ -30,145 +29,133 @@ def main():
         wt_cnt = {}
         mt_cnt = {}
 
-        # file handler for wt and mt sequences
-        fh_wt = {}
-        fh_mt = {}
-
-        epilens = options.lengths.split(',')
-        epilens = [eval(i) for i in epilens]
+        epilens = [eval(i) for i in options.lengths.split(',')]
```

Ruff (0.15.13)

[error] Use of possibly insecure function; consider using `ast.literal_eval`

(S307)